config/areas/system/views.php

```diff
@@ -21,7 +22,13 @@
 				[
 					'label' => I18n::translate('version'),
 					'value' => $kirby->version(),
-					'link'  => 'https://github.com/getkirby/kirby/releases/tag/' . $kirby->version()
+					'info'  => $updateStatus?->label(),
+					'link'  => (
+						$updateStatus ?
+						$updateStatus->url() :
+						'https://github.com/getkirby/kirby/releases/tag/' . $kirby->version()
```

Comment:
This won't work because `$updateStatus->url()` can also return `null` (for "no URL"). We only need the fallback if the whole `$updateStatus` object is `null`.